index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
-	a reasonable opportunity to review the document,
-	who has provided comments,
-	the record of requests to and responses from reviewers,
-	especially <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">W3C Horizontal Groups</a> [[CHARTER]]
-	and groups identified as dependencies in the charter
-	or identified as <a href="https://www.w3.org/2001/11/StdLiaison.html">liaisons</a> [[LIAISON]],
-	and seek evidence of clear communication to the general public
-	about appropriate times and which content to review
-	and whether such reviews actually occurred.
+	The objective of <dfn id=dfn-wide-review export>wide review</dfn> is to seek
+	feedback on a specification from the entire web community, including the general
+	public. The goal is to make the specification as robust as possible before it
+	becomes a [=W3C Recommendation=]. 
 
-	For example,
-	inviting review of new or significantly revised sections published in Working Drafts,
-	and tracking those comments
-	and the [=Working Group=]'s responses,
-	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
-	as well as the general public,
-	especially those affected by this specification,
-	a proposal to enter [=Candidate Recommendation=]
-	(for example in approximately 28 days).
-	By contrast a generic statement in a document
-	requesting review at any time
-	is likely not to be considered as sufficient evidence
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
+	Wide review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
+
+	As a specification progresses along the Recommendation Track, Working Groups
+	<em class="rfc2119">should</em> solicit review from some or all of the following
+	stakeholders: 
+
+	<ul>
+		<li>The Working Group responsible for the specification</li>
+		<li>The horizontal review groups (see 6.2.3.2)</li>
+		<li>The Working Groups and organisations documented in the Working Group's charter</li>
+		<li>The general public</li>
+		<li>Other relevant standards organizations</li>
+	</ul>
+
+	The Director <em class="rfc2119">must</em> consider evidence of wide review before
+	approving transitions. Appropriate evidence may include: 
```

Comment:
Is this "may include" intended to be a closed whitelist of the only appropriate kinds of evidence, or an open list of which these are two examples?

Reply:
The latter.

Reply:
Thanks @LJWatson it would be good to find an editorial way to make that clearer.